src/NzbDrone.Core/Notifications/Stash/StashProxy.cs

```diff
@@ -23,6 +23,20 @@ public void Update(StashSettings settings, string path)
             var generatePreviews = settings.GeneratePreviews ? "true" : "false";
             var cleanPath = path.ToJson();
 
+            var metadataIdentifyQuery = "";
+            if (settings.MetadataIdentify) 
```

Comment:
```suggestion
            if (settings.MetadataIdentify)
```

Reply:
Sorry only just spotted that error in the azure build log